### PR TITLE
Remove config.json file

### PR DESCRIPTION
**Pull request by wcandillon**

### CODE REVIEW

**Comment by wcandillon** [comment]
@dknochen an good honest refactoring.

File: tasks/templates.js [hunk 1]
```diff
@@ -8,40 +8,23 @@ var $ = require('gulp-load-plugins')();
 var _ = require('lodash');
 var mkdirp = require('mkdirp');
 var path = require('path');
+var Mustache = require('mustache');
 
 var Config = require('./config');
 
-var loadCredentials = function(){
+gulp.task('templates:config', [ 'crypt:decrypt' ], function(){
```
Comment (dknochen) [inline tasks/templates.js]: I think, there is no reason any more why this task should stay in templates.js. 
We should move this rest into config.js as well.

**Comment by dknochen** [comment]
@wcandillon please see: https://github.com/28msec/cellstore/pull/794

**Comment by wcandillon** [comment]
@dknochen Nice catch! That actually optimizes a lot.

**Comment by wcandillon** [comment]
@dknochen back to you

**Pull request merged**